next-themes/src/index.tsx

```diff
@@ -49,6 +50,17 @@ const Theme = ({
   const [resolvedTheme, setResolvedTheme] = React.useState(() => theme === 'system' ? getSystemTheme() : theme)
   const attrs = !value ? themes : Object.values(value)
 
+  const isThemeCurrentTheme = (theme: string) => {
+    const attributes = Array.isArray(attribute) ? attribute : [attribute]
+    if (!attributes) return false
```

Comment:
```suggestion
```

I think you can remove this line since `attributes` is always an array by this point which is always truthy.